app/enterprise/2.1.x/kong-manager/administration/workspaces/add-workspace.md

```diff
@@ -47,8 +47,8 @@ regardless of letter case. For example, naming one
 **Workspace** "Payments" and another one "payments" will
 create two different workspaces that appear identical.
 
-⚠️ **WARNING**: Do not name Workspaces the same as these major
-routes in Kong Manager:
+⚠️ **WARNING**: Do not name Workspaces the same as these major API names (paths) 
+in Admin API:
 
 * Admins
```

Comment:
To add this list to the alert block, add `>` to the blank line before the list, and before the first item in the list.

```
>
> * Admins
```